test/unit/deletion_test.rb

```diff
@@ -61,7 +61,7 @@ class DeletionTest < ActiveSupport::TestCase
 
   teardown do
     # This is necessary due to after_commit not cleaning up for us
```

Comment:
> on rails 4.2+ we dont need to disable txn fixtures to test after_commits

I am pretty sure support for testing `after_commit` is [released with rails 5](http://edgeguides.rubyonrails.org/5_0_release_notes.html#active-record-notable-changes).